java/src/main/native/src/ColumnVectorJni.cpp

```diff
@@ -286,7 +286,7 @@ JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnVector_ifElseSS(JNIEnv *env, j
 JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnVector_reduce(JNIEnv *env, jclass,
                                                                 jlong j_col_view,
                                                                 jlong j_agg,
-                                                                jint j_dtype) {
+                                                                jint j_dtype, jint scale) {
```

Comment:
We need to use the scale not just pass it in.

Reply:
Had thought of doing this in follow on PR. Will make the modifications here itself.

Reply:
done